code/modules/food_and_drinks/food/condiment.dm

```diff
@@ -227,6 +228,13 @@
 	list_reagents = list("mayonnaise" = 50)
 	possible_states = list()
 
+/obj/item/reagent_containers/condiment/yoghurt
```

Comment:
You may want to change this from yoghurt to yogurt to follow "All new var/proc names should use the American English spelling of words. This is for consistency with BYOND."
```suggestion
/obj/item/reagent_containers/condiment/yogurt
```